python/lsst/afw/math/boundedField.cc

```diff
@@ -22,14 +22,36 @@
 
 #include <pybind11/pybind11.h>
 //#include <pybind11/operators.h>
-//#include <pybind11/stl.h>
+#include <pybind11/stl.h>
+
+#include "numpy/arrayobject.h"
+#include "ndarray/pybind11.h"
+#include "ndarray/converter.h"
+
+#include "lsst/afw/geom/Point.h"
+#include "lsst/afw/math/BoundedField.h"
 
 namespace py = pybind11;
+PYBIND11_DECLARE_HOLDER_TYPE(MyType, std::shared_ptr<MyType>);
```

Comment:
Not needed anymore after switch to latest pybind11 version. But feel free to leave in for now. I will take it out everywhere later.